app/services/content_item_publisher.rb

```diff
@@ -1,10 +1,48 @@
 class ContentItemPublisher
   def publish(flow_presenters)
+    find_smart_answers_without_start_pages(flow_presenters)
     flow_presenters.each do |smart_answer|
-      content_item = FlowContentItem.new(smart_answer)
-      Services.publishing_api.put_content(content_item.content_id, content_item.payload)
-      Services.publishing_api.publish(content_item.content_id, 'minor')
+      content_item = assign_content_item(smart_answer)
+      if content_item
+        Services.publishing_api.put_content(content_item.content_id, content_item.payload)
+        Services.publishing_api.publish(content_item.content_id, 'minor')
+      end
     end
+    republish_start_page
+  end
+
+  def find_smart_answers_without_start_pages(flow_presenters)
+    @start_page_less_smart_answer = flow_presenters.select { |smart_answer| smart_answer.slug == "part-year-profit-tax-credits/y" }.first
+    @start_page_smart_answer = flow_presenters.select { |smart_answer| smart_answer.slug == "part-year-profit-tax-credits" }.first
+  end
+
+  def assign_content_item(smart_answer)
+    case smart_answer
+    when @start_page_less_smart_answer
```

Comment:
Happy to rename @start_page_less_smart_answer to @smart_answer_without_start_page if it's more readable - will need to also rename a few classes and variables.

Don't want to turn smart_answer into an instance variable however as I'd rather keep its scope to just that method.